test/note_test.rb

```diff
@@ -2,19 +2,52 @@
 
 class TestNote < MiniTest::Test
   def setup
-    @frequency   = 220.to_f
-    @duration    = 2.0
-    @sample_rate = 22050
-    @note = MusicTheory::Note.new(frequency: @frequency, duration: @duration)
+    @test_samples = eval(File.read('test/fixtures/note_samples_array'))
+    @basic_note = MusicTheory::Note.new
```

Comment:
For this to work with `test_samples_are_identical` relies on the default frequency and durations never changing. May as well keep them explicit.